hyperledger-labs/perun-node
# Change non-collaborative to collaborative close in tests

#### Description

Previously non-collaborative close was used in integration tests because of a bug in the go-perun framework where no events were received during a collaborative close. This was fixed in the updated version of go-perun (v0.5.0). Hence use it for faster tests.


##### Category




Implementation Task

##### Relevant issue


Resolves #144 

#### Testing


##### Steps to run the tests


All existing tests should pass. `Test_Integ_Role` and `Test_Integ_Server` will run faster (by approx 10s each) as we opt for the collaborative close instead of non-collaborative close. (10s is the challenge duration used in tests).

#### Checklist
